lib/namae/parser.y

```diff
@@ -246,13 +259,21 @@ require 'strscan'
     input.rest.strip.split(/\s+/)[0] =~ suffix
   end
 
+  def pronoun?
+    !@suffices.zero? || will_see_pronoun?
+  end
+
+  def will_see_pronoun?
```

Comment:
I'll have to look at this in more detail. I guess you added this because tests were failing or was there another reason?